frontend/src/metabase/css/core/colors.module.css

```diff
@@ -103,7 +103,6 @@
     #000
   );
   --mb-color-syntax-parameters: var(--mb-base-color-mango-30);
-  --mb-color-notebook-step-bg: var(--mb-base-color-orion-10);
```

Comment:
Instead of using orion-10, we use dynamic CSS generation for the main Metabase app as well. See screenshot on the PR.